packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -180,17 +185,31 @@ export function NetworksAndSignersProvider(
     }
   }, [isTosAccepted, openConnectModal])
 
+  const invocationCounter = useRef(0)
+
   // TODO: Don't run all of this when an account switch happens. Just derive signers from networks?
   const update = useCallback(async () => {
-    if (!address || !chain) {
+    if (
+      !address ||
+      !chain ||
+      typeof isConnectedToArbitrum === 'undefined' ||
+      typeof isConnectedToOrbitChain === 'undefined'
```

Comment:
This added check seems unnecessary - Please correct me if I am wrong - when `chain` is undefined, wouldn't both `isConnectedToArbitrum` and `isConnectedToOrbitChain` be typeof undefined by the logic defined in their hooks?

Reply:
yes I think we can remove